nss-day-cohort-47/tabloidfullstack-bingo
# Ad post details

# Description
User can now view post details
Fixes # (issue)
## Type of change

# Testing Instructions
Log in as user => click on 'post details' => Post details should include author, title, contents, category, and publish date
# Checklist:
